src/Workspaces/TestSourceGenerator/Constants.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.CodeAnalysis.TestSourceGenerator
+{
+    public class Constants
+    {
+        public const string GeneratedEnglishClassName = "HelloWorld";
+        public const string GeneratedSpanishClassName = "HolaMundo";
+    }
```

Comment:
Hmm, was this just a general refactoring or was it needed for some specific reason? I wasn't sure if we'd eventually need/want different generators in this project...

Reply:
I didn't realize when writing this this was being used also for the other test. That's kinda terrifying for me.

Reply:
I'm just abusing the fact that the test infra has access to a DLL that we can control. This isn't used as a generator at all.

Reply:
As discussed privately: I think we can just have the integration test add itself as an assembly and you could put the needed contents there; the only reason I needed the generator project to do specific magic here was because of TFM limitations: the compiler demands generators implement netstandard2.0 and I couldn't have a test project targeting that which was also targeting VS dependencies.